validphys2/src/validphys/core.py

```diff
@@ -618,6 +626,24 @@ def load(self):
     def load_commondata(self):
         return [d.load_commondata() for d in self.datasets]
 
+
+    def load_commondata_instance(self):
+        """
+        Given Experiment load list of validphys.coredata.CommonData
+        objects with cuts already applied
+        """
+        commodata_list = []
+        for dataset in self.datasets:
+            if dataset.cuts is None:
+                commodata_list.append(dataset.commondata.load_commondata_instance())
+            else:
+                commodata_list.append(
+                    dataset.commondata.load_commondata_instance().with_cuts(
+                        dataset.cuts.load()
+                    )
```

Comment:
I would take `dataset.commondata.load_commondata_instance()` outside of the if condition since it will ease future debugging and it is used in both branches.